vlog.css

```diff
@@ -0,0 +1,166 @@
+
+
+
+
+html {
+height: 300px;
+width: 700px; 
+background-color: #28282B;
+}
+
+.main-menu {
+  background-size: cover;
+  background-color: #181818;
+  height: 50em; 
+  width: 10em;
+  display: flex; 
+  float: left ; 
+  flex-direction: column; 
+  margin-left:-8px;
+  margin-top: -8px;
+
+}
+
+.headline {
+  text-align: center;
+  color: white;
+  margin-top: -40px;
+}
+
+.small_headline {
+  text-align: center;
+  color: grey;
+  opacity: 0.7;
+  font-size: 12px;
+  margin-top: -20px;
+  padding: 20px;
+}
+
+
+.vertical-menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-top: 80px;
+
+  background-color: #181818;
+  color: #909096; 
+  padding: 20px; 
+  text-decoration: none; 
+  text-align: center;
+  border: 0.2px solid #282828;
+  Font: 16px "Open-Sans", sans-serif;
+}
+
+a{
+  text-decoration: none;
+  color: #909096;
+}
+
+.About{
+  text-align: center;
+  border: 0.2px solid #282828;
+  Font: 16px "Open-Sans", sans-serif;
```

Comment:
let's move it to the body level

Reply:
Done

Reply:
I moved the "Font" line to the body level, but can't do the same with text-align right now.  Or it will change all website text orientation